.npmrc

```diff
@@ -0,0 +1 @@
+tag-version-prefix=
```

Comment:
`npm version`'s default is to use the `v` prefix on tags, but since we haven't been doing that (I don't happen like it, personally), this config lets us stay consistent.